pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter.go

```diff
@@ -140,6 +140,16 @@ func (admitter *VMsAdmitter) authorizeVirtualMachineSpec(ar *v1beta1.AdmissionRe
 		}
 	}
 
+	if len(causes) > 0 {
+		return causes, nil
+	}
+
+	causes = validateNoModificationsDuringRename(ar, vm)
+
+	if len(causes) > 0 {
```

Comment:
this statement seems unnecessary

Reply:
I put it there so when another person adds code to this function, they can just add code above the return line, and won't have to deal with the error handling of my code. It feels more comfortable to me this way, we can discuss further if it still bothers you

Reply:
If that is your intention, it should probably be made explicit with a comment.  Obviously, this is not a huge deal but it looks kind of weird otherwise.

Reply:
Sounds fair, I will add a comment to make it more clear, thanks!